superset-frontend/spec/javascripts/explore/AdhocFilter_spec.js

```diff
@@ -153,6 +153,27 @@ describe('AdhocFilter', () => {
     });
     // eslint-disable-next-line no-unused-expressions
     expect(adhocFilter5.isValid()).toBe(true);
+
+    const adhocFilter6 = new AdhocFilter({
+      expressionType: EXPRESSION_TYPES.SIMPLE,
+      subject: 'value',
+      operator: '==',
+      comparator: 1,
+      clause: CLAUSES.WHERE,
+    });
+    // eslint-disable-next-line no-unused-expressions
+    expect(adhocFilter6.isValid()).toBe(true);
```

Comment:
This specific test is the one that proves my point, tested that it fails on `master`

Reply:
Should this be `comparator: 0`?

Reply:
Both 0 and 1 should be true, happy to add an entry for `0`, especially since it's falsy it's good to secure assumptions

Reply:
Looking at the old code I originally thought it was the falsiness that was causing the problem. My bad. But probably a good idea to add a test for that, just in case.